stdlib/public/SDK/AVFoundation/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ add_swift_library(swiftAVFoundation ${SWIFT_SDK_OVERLAY_LIBRARY_BUILD_TYPES} IS_
   TARGET_SDKS OSX IOS IOS_SIMULATOR TVOS TVOS_SIMULATOR
   SWIFT_COMPILE_FLAGS "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}"
   LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
-  SWIFT_MODULE_DEPENDS Foundation CoreMedia QuartzCore
-  SWIFT_MODULE_DEPENDS_OSX AppKit
+  SWIFT_MODULE_DEPENDS Foundation CoreMedia
```

Comment:
Actually, this is used all over in ``stdlib/private`` and I'm only changing the ``public`` directory for now.